sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -288,10 +288,9 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
 
   test("SPARK-32234 read ORC table with column names all starting with '_col'") {
     Seq("native", "hive").foreach { orcImpl =>
-      Seq("false", "true").foreach { vectorized =>
-        withSQLConf(
-          SQLConf.ORC_IMPLEMENTATION.key -> orcImpl,
-          SQLConf.ORC_VECTORIZED_READER_ENABLED.key -> vectorized) {
+      withSQLConf(
+        SQLConf.ORC_IMPLEMENTATION.key -> orcImpl) {
+        withAllNativeOrcReaders {
```

Comment:
Ur, @beliefer . This is technically correct, but logically misleading again. When `orcImpl=hive`, we are not testing `nativeOrcReaders`.

Reply:
I'll merge this PR without this part.

Reply:
@dongjoon-hyun Thank you for the reminder.